src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1388,16 +1425,17 @@ namespace winrt::TerminalApp::implementation
             auto selectedIndex = tabView.SelectedIndex();
 
             // Unfocus all the tabs.
-            for (auto tab : _tabs)
+            for (uint32_t idx = 0; idx < _tabs.Size(); ++idx)
             {
+                auto tab{ _GetStrongTabImpl(idx) };
                 tab->SetFocused(false);
             }
```

Comment:
minor nit: weird because GetStrongTabImpl takes an _index_ not a tab, but these are better-formed as `for(auto tab:_tabs)` like in the original code.

I also don't care as much. :)

Reply:
I also was nitting about this, so I _sort of_ wanted to make `GetStrongTabImpl` take a tab instead. Then I realized that the majority of the code passes in an index and so if it took a Tab, `GetAt` would be littered everywhere instead. So, in my opinion this could be the lesser of the two evils. 🤷‍♂

Reply:
You could, in theory, provide an overload. One takes a const uint32_t and one takes a `::winrt::TerminalApp::Tab` 😀

Reply:
🤔 Do you think an overload for this function would be overkill?

Reply:
up to you. it would clean our iterators up again, but also I don't hate this